packages/tests/integration/src/entities/codegen/LargePublisherCodegen.ts

```diff
@@ -104,13 +115,17 @@ export interface LargePublisherGraphQLFilter extends PublisherGraphQLFilter {
 export interface LargePublisherOrder extends PublisherOrder {
   sharedColumn?: OrderBy;
   country?: OrderBy;
+  spotlightAuthor?: AuthorOrder;
 }
 
 export interface LargePublisherFactoryExtras {
 }
 
 export const largePublisherConfig = new ConfigApi<LargePublisher, Context>();
 
+largePublisherConfig.addRule(newRequiredRule("spotlightAuthor"));
+largePublisherConfig.addRule("spotlightAuthor", mustBeSubType("spotlightAuthor"));
```

Comment:
Little surprised the `mustBeSubType` rule showed up, given `Author` doesn't have any subtypes

Reply:
hmm yeah, why is this not breaking any tests? 🤔

Reply:
I think `mustBeSubType` just gets the "what should I `instanceof` against?" from the runtime metadata, so it's probably just adding a "passes but technically unnecessary" `spotlightAuthor instanceof Author` check. We could probably just leave it.